main.py

```diff
@@ -1,10 +1,56 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_data = json.load(players_file)
+
+    for player in players_data:
+        player_nickname = player
+        player_email = players_data[player]["email"]
+        player_bio = players_data[player]["bio"]
+        race = players_data[player]["race"]
+
+        if Race.objects.filter(name=race["name"]).exists():
+            player_race = Race.objects.get(name=race["name"])
+        else:
+            player_race = Race.objects.create(
+                name=race["name"],
+                description=race["description"]
+            )
+
+        skills = race["skills"]
+
+        for skill in skills:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=player_race
+                )
```

Comment:
Use get_or_create for this action
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create